fastlane/Fastfile

```diff
@@ -0,0 +1,286 @@
+require 'yaml'
+require_relative 'version'
+
+desc "Commit version file"
+private_lane :github_commit_version do |options|
+  require 'openssl'
+  require 'jwt'
+  require 'octokit'
+  require 'base64'
+
+  branch = git_branch() # Commit to current branch
+  version_yml_path = options[:version_yml_path]
+  repo = options[:repo]
+  new_version_code = options[:new_version_code]
+  new_version_name = options[:new_version_name]
+  app_id = options[:app_id]
+  private_key = OpenSSL::PKey::RSA.new(Base64.decode64(options[:private_key]))
+
+  payload = {
+      iat: Time.now.to_i,
+      exp: Time.now.to_i + (5 * 60),
+      iss: app_id
+  }
+
+  jwt = JWT.encode(payload, private_key, 'RS256')
+  client = Octokit::Client.new(:bearer_token => jwt)
+
+  installation = client.find_repository_installation(repo)
+  token = client.create_app_installation_access_token(installation.id).token
+  client = Octokit::Client.new(:access_token => token)
+
+  version_yml_content = client.contents(repo, path: version_yml_path, query: {ref: branch})
+  version_yml = YAML.load(Base64.decode64(version_yml_content.content))
+  version_yml['version_code'] = new_version_code
+  version_yml['version_name'] = new_version_name
+
+  client.update_contents(repo,
+                         version_yml_path,
+                         "Bump version name to #{new_version_name} and version code to #{new_version_code}\n[skip ci]",
+                         version_yml_content.sha,
+                         YAML.dump(version_yml),
+                         :branch => branch)
+end
+
+desc "Upload source map to Sentry"
+private_lane :generic_sentry_upload do |options|
+  ensure_env_vars(
+      env_vars: ['SENTRY_AUTH_TOKEN']
+  )
+
+  next_version = next_version()
+  version_code = next_version[:new_version_code]
+  version_name = next_version[:new_version_name]
+  package_name = options[:package_name]
+  sourcemap_path = options[:sourcemap_path]
+
+  sentry_create_release(
+      version: version_name,
+      app_identifier: package_name,
+      finalize: true
+  )
+
+  sentry_upload_sourcemap(
+      version: version_name,
+      app_identifier: package_name,
+      dist: version_code.to_s,
+      sourcemap: sourcemap_path,
+      rewrite: true
+  )
+end
+
+desc "Bump version file in Git repository"
+lane :github_bump_version do
+  ensure_env_vars(
+      env_vars: ['DELIVERINO_PRIVATE_KEY']
+  )
+
+  github_commit_version(
+      app_id: 59249,
+      private_key: ENV['DELIVERINO_PRIVATE_KEY'],
+      repo: "Integreat/integreat-react-native-app",
+      version_yml_path: 'version.yml',
+      **next_version
```

Comment:
what does `next_version` reference to here? What's the `**` operator?

Reply:
Ruby syntax, ** is spread and next_version is a function call